portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -2,19 +2,19 @@
 
 /** Object that stores comment data. */
 public class Comment {
+  private String blobKeyString;
   private String commenterEmail;
   private String commenterName;
   private long id;
-  private String imageUrl;
   private String text;
   private long timestamp;
 
-  public Comment(String commenterEmail, String commenterName, long id, String imageUrl, String text,
-      long timestamp) {
+  public Comment(String blobKeyString, String commenterEmail, String commenterName, long id,
+      String text, long timestamp) {
+    this.blobKeyString = blobKeyString;
     this.commenterEmail = commenterEmail;
     this.commenterName = commenterName;
     this.id = id;
-    this.imageUrl = imageUrl;
```

Comment:
Next time I think you can put removing image in a separate PR?

Reply:
Ack - will do.